packages/components/src/components/vertical-tab/vertical-tab.jsx

```diff
@@ -40,7 +52,7 @@ class VerticalTab extends React.Component {
     }
 
     render() {
-        const selected = this.props.list[this.props.modal_index] || this.props.list[0];
+        const selected = this.props.list[this.state.modal_index] || this.props.list[0];
         return (
             <div
                 className={classNames('vertical-tab', {
```

Comment:
Please prefix all classnames in this file with `dc-` (with updates to their respective stylesheet).